ATTENUATOR_FLASHING.md

```diff
@@ -1,151 +1,151 @@
+
+📝 **Note:** When using this firmware there will be additional debug messages enabled for the device. Therefore, this firmware image may also be used to help debug WiFi issues by checking the output via the Arduino IDE's serial console. Be sure to set the baud rate to 112500 to view the output correctly.
+
+---
+
+## Software Development Requirements
+
+To build or edit the code for this device you must have an ArduinoIDE environment set up similar to what is needed for the pack/wand software. Download and install the Arduino IDE 2.x or higher. This will be used to compile and upload the code to your Proton Pack and Neutrona Wand.
+
+[Arduino IDE 2.x](https://www.arduino.cc/en/software)
+
+The following libraries are required to be installed. All but the MillisDelay library can be found within the Arduino Library Manager with the app. Go to `Sketch -> Include Library -> Manage Libraries...` to access the Library Manager. Search for the libraries by name and install the latest version available.
+
+### Common Libraries (Nano+ESP32)
+
+- **FastLED** by Daniel Garcia
+- **ezButton** by ArduinoGetStarted.com
+- **simple ht16k33 library** by lpaseen
+- **SerialTransfer** by PowerBroker2
+- **SafeString** by Matthew Ford
+
+### Additional ESP32 Libraries
+
 - **ArduinoJSON** by Benoit Blanchon
 - **AsyncTCP** by dvarrel
-- **ESP Async WebServer** by Me-No-Dev
-- **ElegantOTA** by Ayush Sharma `See Below`
-
-To build for the ESP32 hardware you will need to use the `Boards Manager` to install the `esp32 by Expressif Systems` package. When selecting a board for compilation and upload, simply use the board `ESP32 Dev Module` for satisfactory results. For reference, the FQBN for builds is "esp32:esp32:esp32".
-
-### ElegantOTA
-
-The ElegantOTA library must be enabled to utilize the Asynchronous Web Server.
-
-1. Go to your Arduino libraries directory, typically found at `~/Documents/Arduino/library`.
-1. Open the `ElegantOTA` folder and then open the `src` folder
-1. Locate the `ELEGANTOTA_USE_ASYNC_WEBSERVER` macro in the `ElegantOTA.h` file, and set it to 1:
-	`#define ELEGANTOTA_USE_ASYNC_WEBSERVER 1`
-1. Save the changes to the `ElegantOTA.h` file.
-
-**Alternative:**
-
-A fork of the `ElegantOTA` library can be downloaded from GitHub which contains the above modifications for use with the ESPAsyncWebServer package. Download the code as a zip use the `Sketch -> Add .ZIP Library` option to import the downloaded file.
-[https://github.com/DustinGrau/ElegantOTA.git](https://github.com/DustinGrau/ElegantOTA.git)
-
-No further configuration is needed for this library.
+- **ESP Async WebServer** by Mathieu Carbou

```

Comment:
What's interesting is that in ArduinoIDE it gives the name "me-no-dev" which I wanted to keep to avoid confusion. Though the "More Info" link takes you to the GitHub repo by mathieu Carbou and reflects the expected 2.5.1 version. I don't know if it's more confusing to change this or leave it as what ArduinoIDE shows.

Reply:
That's odd; when I searched for it in the Library Manager the name came up as Mathieu Carbou. Screenshot attached.

![espasync](https://github.com/gpstar81/GPStar-proton-pack/assets/11485516/958c6e32-bec3-409d-8690-f5f33208ce58)